routers/api/v2/users.go

```diff
@@ -25,7 +28,7 @@ func (r *apiV2Router) Login(ctx *gin.Context) {
 		Email    string `form:"email"`
 		Password string `form:"password"`
 	}
-	ctx.Bind(&req)
+	_ = ctx.Bind(&req)
```

Comment:
Why is this necessary? Does `ctx.Bind(&req)` not work?

Reply:
`ctx.Bind(&req)` does work, I was just getting annoyed by the `Unhandled error` message GoLand was giving me all the time 😅

Reply:
ah okay let's leave it like that then 😄